doc/running.md

```diff
@@ -62,7 +122,9 @@ python3 manage.py migrate
 ```sh
 python3 manage.py createsuperuser
 ```
-NB `Username` needs to be your GitHub username
+
+Your `Username` needs to be your GitHub username.
+Your `Auth0 id` needs to be ??? (not working for me yet).
```

Comment:
If you log into Auth0, click on the "Users & Roles" link, then find your own user you'll on the right of your avatar the `user_id:` field, the value will start with `github|`, that's your Auth0 ID and it will have to include the `github|` prefix.

Reply:
I tried exactly that (guessing), but it didn't work. :-( It's probably me, I'll try again.

Reply:
I had the API call to hand that gets a github user ID, so thought I'd share: https://api.github.com/users/davidread So the corresponding Auth0 id is `github|307612`. Whether that's what this field needs, is another matter though.